src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -1972,18 +1972,104 @@
 
 helps['tag create'] = """
 type: command
-short-summary: Create a tag in the subscription.
+short-summary: Create tags on a specific resource.
+long-summary: >
+    The az tag create cmdlet with an id creates or updates the entire set of tags on a resource, resource group or subscription.
```

Comment:
Please quote the command with backticks `<command>` to tell the document renderer to parse it as **code**.
```suggestion
    The `az tag create` command with an id creates or updates the entire set of tags on a resource, resource group or subscription.
```